produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
```

Comment:
Could we add a check here to only do the `UI.input` if `UI.interactive?` to avoid a potential crash? If `UI.interactive?` returns false, we should raise a `UI.user_error` saying that if the shell is not interactive the `:merchant_identifier` option is required. Does that make sense?